server/protocols/jmap/src/main/java/org/apache/james/jmap/model/SetMailboxesResponse.java

```diff
@@ -0,0 +1,61 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+package org.apache.james.jmap.model;
+
+import org.apache.james.jmap.methods.Method;
+import org.apache.james.jmap.model.mailbox.Mailbox;
+
+import com.google.common.collect.ImmutableMap;
+
+public class SetMailboxesResponse implements Method.Response {
+
+    public static Builder builder() {
+        return new Builder();
+    }
+
+    public static class Builder {
+
+        private ImmutableMap.Builder<MailboxCreationId, Mailbox> created;
+
+        private Builder() {
+            created = ImmutableMap.builder();
+        }
+
+        public Builder creation(MailboxCreationId creationId, Mailbox mailbox) {
+            created.put(creationId, mailbox);
+            return this;
+        }
+
+        public SetMailboxesResponse build() {
+            return new SetMailboxesResponse(created.build());
+        }
+
+    }
+
+    private final ImmutableMap<MailboxCreationId, Mailbox> created;
+
+    private SetMailboxesResponse(ImmutableMap<MailboxCreationId, Mailbox> created) {
+        this.created = created;
+    }
+
+    public ImmutableMap<MailboxCreationId, Mailbox> getCreated() {
+        return created;
+    }
+
+}
```

Comment:
as you wish. As said previously, we used to define all the protocol request/response the first time we implement a functionality. I think this is good because it is easier to share work later.
But I'm agree that in the response case it is less necessary than in the request case.